5.5/contrib/common.sh

```diff
@@ -20,13 +20,13 @@ mysql_password_regex='^[a-zA-Z0-9_~!@#$%^&*()-=<>,.?;:|]+$'
 
 function usage() {
   [ $# == 2 ] && echo "error: $1"
-  echo "You must specify following environment variables:"
+  echo "You must either specify the following environment variables:"
   echo "  MYSQL_USER (regex: '$mysql_identifier_regex')"
   echo "  MYSQL_PASSWORD (regex: '$mysql_password_regex')"
   echo "  MYSQL_DATABASE (regex: '$mysql_identifier_regex')"
-  echo "Optional:"
+  echo "Or the following environment variable:"
   echo "  MYSQL_ROOT_PASSWORD (regex: '$mysql_password_regex')"
-  echo "Settings:"
```

Comment:
yes.      just trying to clarify to users that they can set both a ROOT password and a USER password, they aren't mutually exclusive, but at least one is required.